src/ImageSharp/Image/ImageExtensions.cs

```diff
@@ -138,7 +138,7 @@ public static void Save<TPixel>(this Image<TPixel> source, Stream stream, IImage
         /// <param name="source">The source image</param>
         /// <param name="buffer">The buffer to save the raw pixel data to.</param>
         /// <exception cref="System.ArgumentNullException">Thrown if the stream is null.</exception>
-        public static void SavePixelData<TPixel>(this Image<TPixel> source, Span<byte> buffer)
+        internal static void SavePixelData<TPixel>(this Image<TPixel> source, byte[] buffer)
```

Comment:
We could probably expose a `SavePixelData(byte[] buffer, int offset)` variant publicly.

Reply:
i've made it public in the Image-Frame branch